src/theory/uf/equality_engine_types.h

```diff
@@ -351,11 +351,10 @@ struct TriggerInfo {
   Node trigger;
   /** Polarity of the trigger */
   bool polarity;
-  TriggerInfo() {}
+  TriggerInfo() : polarity() {}
```

Comment:
Is there a reason why polarity is not explicitly initialized to false? I know it's the default in this case, but imo it is more readable to add false here.

Reply:
> Is there a reason why polarity is not explicitly initialized to false?
I am taking a guess at the author's intent. So I do not have one.

Changed it to false.